docs/source/tutorials/datalab/workflows.ipynb

```diff
@@ -375,7 +378,10 @@
    "source": [
     "## Find Underperforming Groups in a Dataset\n",
     "\n",
-    "Here we will demonstrate how to use `Datalab` to identify subgroups in a dataset over which the ML model is producing consistently worse predictions than for the overall dataset."
+    "Here we will demonstrate how to use `Datalab` to identify subgroups in a dataset over which the ML model is producing consistently worse predictions than for the overall dataset.\n",
+    "\n",
+    "`Datalab` will automatically find underperforming groups if you provide numerical embeddings and predicted probabilities from any model.\n",
+    "For this section, we'll focus on how to decide which groups to consider, such as by using clustering.\n"
```

Comment:
i suggested changes to this in your other PR, not sure why there were two PRs with the same changes, but please don't overlook / accidentally undo my suggestions when merging this PR

Reply:
I've merged those changes into this branch. Those suggestions have been applied here.